src/Integration/Suppression/SonarQubeIssuesProvider.cs

```diff
@@ -106,15 +107,11 @@ public IEnumerable<SonarQubeIssue> GetSuppressedIssues(string projectGuid, strin
                 return Enumerable.Empty<SonarQubeIssue>();
             }
 
-            string moduleKey = BuildModuleKey(projectGuid);
-            return this.cachedSuppressedIssues.Where(x =>
-                x.FilePath.Equals(filePath, StringComparison.OrdinalIgnoreCase) &&
-                x.ModuleKey.Equals(moduleKey, StringComparison.OrdinalIgnoreCase));
-        }
-
-        private string BuildModuleKey(string projectGuid)
-        {
-            return $"{sonarQubeProjectKey}:{sonarQubeProjectKey}:{projectGuid}";
+            string moduleKey = $"{sonarQubeProjectKey}:{sonarQubeProjectKey}:{projectGuid}";
```

Comment:
This is the format of the legacy module key, so we need to match against this for old versions of SQ and newer versions where the module keys haven't been migrated to the new format.
However, I thought that the module key in the new module-less world will just be the project key for the root project. Don't we need to match against that value too?